### PR TITLE
Add icons to facia cards in email

### PR DESCRIPTION
**Pull request by joelochlann**
Adds in the video/audio/photo/quote icons. It made sense to ditch the `float-left` class on the icon images and make them inline - I checked that this doesn't affect the existing icons in the article emails (the photo icon on image captions and twitter/email icons on the byline)

picture 333
picture 338
picture 337
picture 335
picture 334

@superfrank @desbo

### CODE REVIEW

**Comment by sndrs** [comment]
you've got changes to `yarn.lock` but no changes to `package.json`, looks like you may need to revert them and merge master https://theguardian.slack.com/archives/dotcom/p1480008240001843

File: common/app/views/support/EmailHelpers.scala [hunk 1]
```diff
@@ -50,7 +50,7 @@ object EmailHelpers {
   }
 
   def icon(name: String) = Html {
-    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="float-left icon icon-$name">"""
+    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="icon icon-$name">"""
```
Comment (sndrs) [inline common/app/views/support/EmailHelpers.scala]: any reason not to use inlined svgs?

Comment (joelochlann) [inline common/app/views/support/EmailHelpers.scala]: sadly SVGs don't work in the majority of email clients

Comment (sndrs) [inline common/app/views/support/EmailHelpers.scala]: wow crazy, it seems like that would be _harder_ to implement than support! fair enough!

Comment (joelochlann) [inline common/app/views/support/EmailHelpers.scala]: yeah it is mad, it seems like Gmail goes to the effort of actually stripping out SVGs from the email HTML before allowing it to be rendered in someone's inbox!

**Comment by joelochlann** [comment]
Thanks @sndrs, I've reverted the `yarn.lock` change and rebased off of master

**Pull request merged**